src/TextTruncate/TextTruncate.tsx

```diff
@@ -11,12 +12,14 @@ const propTypes = {
   showLessLabel: PropTypes.string,
   lines: PropTypes.number,
   maxCharsShown: PropTypes.number,
+  richText: PropTypes.bool,
```

Comment:
I have a slight preference for `useRichText` to make it more obvious that it's a boolean

Reply:
+1

Reply:
Done!